www/assets/js/databrowser.js

```diff
@@ -162,20 +162,33 @@ browser = simply.app({
     }),
     keyboard: {
         //@TODO: keyboard definition should be in spreadsheet.js, and referenced here
+        //@TODO: e.target.matches might be simplified using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some
         spreadsheet: {
             "ArrowDown": (e) => {
+                if(e.target.matches("input") || e.target.matches("textarea" || e.target.matches("select"))){
```

Comment:
korter:
e.target.matches('input,textarea,select')
single quotes hebben mijn voorkeur, dan zie je direct dat er geen variabelen in gebruikt worden.